scheduler/src/cook/mesos/api.clj

```diff
@@ -716,16 +720,51 @@
   "Takes the executor-id->sandbox-directory from the agent json and constructs a URL to query it. Hardcodes fun
    stuff like the port we run the agent on. Users will need to add the file path & offset to their query. Refer to
    the 'Using the output_url' section in docs/scheduler-rest-api.asc for further details."
-  [fid agent-hostname executor-id]
+  [framework-id agent-hostname executor-id sandbox-location]
```

Comment:
Why not `sandbox-directory`? Also, should we add a comment to the docstring about why this may or may not be `nil`?